package.json

```diff
@@ -69,7 +69,7 @@
     "trim": "~1.0.1",
     "browserslist": "^4.16.5",
     "ws": "^7.4.6",
-    "css-what": "^5.0.1",
+    "css-what": "^6.1.0",
```

Comment:
Do you happen to know if this resolution is still needed?

Reply:
Nice catch! Looked into it, and removing doesn't cause new security issues (checked with `yarn audit`) and tests still pass.